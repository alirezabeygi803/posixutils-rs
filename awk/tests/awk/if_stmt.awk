BEGIN {
    if (2 > 4)
        print "true branch"
    else
        print "false branch"

    if (2 < 4)
        print "true branch"
    else
        print "false branch"

    if (1)
        if (0)
            {}
        else
            print "false branch"

}