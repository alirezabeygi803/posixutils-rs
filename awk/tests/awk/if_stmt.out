false branch
true branch
false branch
